plugins/registry/handler.go

```diff
@@ -138,17 +147,17 @@ func (rh *registryHandler) login(ctx context.Context, p registryHandlerParams) {
 	)
 	if err != nil {
 		logger.Error(err.Error())
-		writeStatusInternalError(req, w)
+		writeStatus(req, w, "13", "INTERNAL", "")
```

Comment:
I think it's more readable if we pass the http status and we do the mapping (we don't even need to map every status for now, only the ones we use) to gRPC in the function. It produces a smaller method signature and devs tend to be more used to `http.Status`